src/sparseml/pytorch/utils/helpers.py

```diff
@@ -137,6 +137,15 @@ def default_device() -> str:
     return "cuda:{}".format(",".join(device_ids))
 
 
+def use_single_gpu(device: str) -> str:
```

Comment:
Potentially useful, related to the open question in to the comment left in `exporters.py`